### PR TITLE
chore: update mysql yaml port

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: yeya24 <yb532204897@gmail.com>

### CODE REVIEW

File: manifests/quick-start-mysql.yaml [hunk 1]
```diff
@@ -555,7 +555,7 @@ spec:
         image: mysql:8
         name: main
         ports:
-        - containerPort: 5432
```
Comment (alexec) [inline manifests/quick-start-mysql.yaml:558]: that is very wrong! - but I think it must be changed in a different file - `mysql-deployment.yaml` and then run `make manifests`?

Comment (yeya24) [reply manifests/quick-start-mysql.yaml:558]: Done, PTAL!

**Comment by alexec** [comment]
Sorry - you need to to sign CLA before merge.

**Comment by yeya24** [comment]
> Sorry - you need to to sign CLA before merge.

Done

**Pull request merged**